ckanext/data_qld/command_functions.py

```diff
@@ -154,3 +155,112 @@ def send_email_dataset_overdue_notification():
     log.info('Started command SendEmailDatasetOverdueNotification')
     resource_freshness_helpers.process_email_notification_for_dataset_overdue()
     log.info('Finished command SendEmailDatasetOverdueNotification')
+
+
+def update_missing_values():
+    '''
+    Update datasets to set default values for metadata
+    '''
+    context = {'session': model.Session}
+    site_user = get_action('get_site_user')({'ignore_auth': True}, {})
+    context['user'] = site_user['name']
+
+    def _get_packages():
+        return model.Session.query(model.Package).filter(model.Package.state == 'active').all()
+
+    def _update_package(package_dict):
+        # Set some defaults
+        try:
+            print('Updating package {0}'.format(package_dict['id']))
+            get_action('package_patch')(context, package_dict)
+            return True
+        except Exception as ex:
+            print('Package exception: %s' % ex)
+            return False
+
+    def _update_resource(res_dict):
+        # Set some defaults
+        try:
+            print('Updating resource {0}'.format(res_dict['id']))
+            get_action('resource_patch')(context, res_dict)
+            return True
+        except Exception as ex:
+            print('Resource exception: %s' % ex)
+            return False
+
+    def _check_for_package_null_values(package):
```

Comment:
This name is misleading; it's not really checking, it's populating.

Reply:
Updated method names to match functionality